test/utils/Address.test.js

```diff
@@ -1,37 +1,21 @@
-const { constants } = require('openzeppelin-test-helpers');
```

Comment:
@RCYP it looks like you've mistakenly removed some of the tests here during the merge, which were added in #1773

Reply:
Thanks